README.md

```diff
@@ -59,7 +59,7 @@ You most likely may override `image`, `applicationPort` and `environment` for yo
 | `memoryLimits` | Memory limits | `256Mi` |
 | `cpuLimits` | CPU limits | `500m` |
 | `ingressHost` | Host for ingress controller to map the container to. It supports templating, Example : {{.Release.Name}}.service.core-compute-preview.internal| `nil` |
-| `registerAdditionalDns.enabled` | If you want to use this chart as a secondary dependency - e.g. providing a frontend to a backend, and the backend is using primary ingressHost DNS mapping. Note: you will also need to define: `ingressIP: ${INGRESS_IP}` and `consulIP: ${CONSUL_LB_IP}` - this will be populated by pipeline | `false` |
+| `registerAdditionalDns.enabled` | If you want to use this chart as a secondary dependency - e.g. providing a frontend to a backend, and the backend is using primary ingressHost DNS mapping. | `false` |
```

Comment:
does this do anything?

Reply:
still used here, to have an additional ingress https://github.com/hmcts/chart-library/blob/master/library/templates/_ingress.tpl#L23.